[RUMF-648][RUM/Logs] add new cookie configurations (#8500)
DataDog/documentation: content/en/real_user_monitoring/browser/_index.md

| `version`                      | String  | No       | ``              | The application’s version e.g. 1.2.3, 6c44da20, 2020.02.13.                                                       |
| `trackInteractions`            | Boolean | No       | `false`         | Enables [automatic collection of Users Actions][6]                                                                |
| `resourceSampleRate`           | Number  | No       | `100`           | Percentage of tracked sessions with resources collection. `100` for all, `0` for none of them.                    |
| `sampleRate`                   | Number  | No       | `100`           | Percentage of sessions to track. Only tracked sessions send rum events. `100` for all, `0` for none of them.      |
| `silentMultipleInit`           | Boolean | No       | `false`         | Initialization fails silently if Datadog's RUM is already initialized on the page                                 |
| `proxyHost`                    | String  | No       | ``              | Optional proxy URL. See the full [proxy setup guide][7] for more information.                                     |

> | `trackSessionAcrossSubdomains` | Boolean | no       | `false`         | Set to `true` to preserve session across subdomains of the same site. **Configuration must match when using RUM SDK**  |

Reviewer: What does "configuration must match" mean? What things must match each other?

Author: if customers use both rum and logs sdk, the value of this option must be the same in both sdk init, ex:

```js
DD_RUM.init({ ..., trackSessionAcrossSubdomains: true })
DD_LOGS.init({ ..., trackSessionAcrossSubdomains: true })
```

any idea on how to better convey this restriction?

Reviewer: How about "If you use both Logs and RUM SDKs, this config must match." (in both docs pages)

> | `useSecureSessionCookie`       | Boolean | no       | `false`         | Set to `true` to use a secure session cookie. This will disable rum events sending on insecure (non-HTTPS) connections. **Configuration must match when using RUM SDK** |

Reviewer: ```suggestion
| `useSecureSessionCookie`       | Boolean | no       | `false`         | Set to `true` to use a secure session cookie. This will prevent RUM events from sending on insecure (non-HTTPS) connections. **Configuration must match when using RUM SDK** |
```

> | `useCrossSiteSessionCookie`    | Boolean | no       | `false`         | Set to `true` to use a secure cross-site session cookie. This will allow the RUM SDK to run when the site is loaded from another one (ex: via an iframe). Implies useSecureSessionCookie. **Configuration must match when using RUM SDK** |

Reviewer: ```suggestion
| `useCrossSiteSessionCookie`    | Boolean | no       | `false`         | Set to `true` to use a secure cross-site session cookie. This will allow the RUM SDK to run when the site is loaded from another one (for example, in an `iframe`). Implies useSecureSessionCookie. **Configuration must match when using RUM SDK** |
```
